src/actors/lan_discovery.rs

```diff
@@ -28,7 +28,7 @@ pub async fn actor(
 
     loop {
         // Create a UDP socket with the given address.
-        let socket = UdpSocket::bind(format!("0.0.0.0:{}", rpc_port)).await?;
+        let socket = UdpSocket::bind(format!("0.0.0.0:{rpc_port}")).await?;
```

Comment:
I must admit I wasn't aware of this change, introduced in Rust 1.58. At first glance I don't like it but I'm probably just being a curmudgeon ☺️ Let's roll with the changes.

Reply:
I didn't know this neither, I just tried out `cargo clippy --fix` for the first time :D

Reply:
But I like it _a lot_ actually, it reduces the differences to python, which just makes me happy because a) less difference == more easier, and b) having more features of dynamic languages in something as strict as Rust takes the wind out of people's sails when they insist they need these super-duper dynamic features in their language.